src/paps.c

```diff
@@ -42,9 +43,8 @@
 int wcwidth(wchar_t c);
 
 #ifndef M_PI
-#define __USE_XOPEN
+#define M_PI    3.14159265358979323846
 #endif
-#include <math.h>
```

Comment:
Actually, why bother? Let's just define PAPS_PI and get rid of the all the ifdef's.

Reply:
Because you changed the PI definition in https://github.com/dov/paps/commit/054a03277e1230faa149a8521979b617dad34220, the glib.h is enough, math.h not needed.
In addition, pango_attr_insert_hyphens_new requires pango 1.44.